### ISSUE TITLE
md-menu: menu items are overlayed when progress bar is running

### ISSUE THREAD

**Issue by viktorzavadil** [angular/material#12040]
## Bug, Enhancement Request, or Proposal

# Demo and steps to reproduce the issue
[This blank StackBlitz demo](https://stackblitz.com/edit/angularjs-material-blank?file=app%2Fapp.template.html) can be used to create a reproduction that demonstrates your issue.

**Demo URL (required)***:  https://codepen.io/vikiza/pen/WNxzvpJ

#### Detailed Reproduction Steps
1. Open menu when progress bar is running

### Explain the expected behavior
Menu has all items.

### Explain the current behavior
If menu is opened when progress bar is running all items is overlaying.

### List the affected versions of AngularJS, Material, OS, and browsers

- AngularJS: 1.7.9, 1.8.2
- AngularJS Material: 1.1.21, 1.2.1
- OS: iOS 13.5
- Browsers: safari
 
#### Stack Trace
```
```

#### Screenshots
<img width="647" alt="Screen Shot 2020-10-29 at 21 26 35" src="https://user-images.githubusercontent.com/19145428/97628815-84469200-1a2d-11eb-968c-c6b0b34cd6d3.png">

**Comment by Splaktar** [comment]
Using the same CodePen (in Full Page or Editor view), I'm not able to reproduce this on an iPad device with iOS 14.1 or in a simulator with 
- iPhone 12 iOS 14.1
- iPhone 11 iOs 14.1
- iPhone 8 iOS 14.1
- iPad Pro 2nd gen 11" iOS 14.1

**Comment by viktorzavadil** [comment]
Yeah, unfortunetally problem is reproducable on iOS 13.5 only. Menu is working on iOS 14 (for me).
